beamer/tests/contracts/test_request_manager.py

```diff
@@ -1418,3 +1418,19 @@ def test_transfer_limit_requests(request_manager, token):
     make_request(request_manager, token, requester, requester, transfer_limit + 1)
 
     assert token.balanceOf(requester) == 0
+
+
+def test_min_lp_fee(request_manager, token, chain_params, token_params, lp_margin_ppm):
+    target_chain_params = chain_params[0], int(3e12), 750_000
+    request_manager.updateChain(999, *target_chain_params)
+
+    source_chain_cost = (1_000_000 - chain_params[2]) * chain_params[1]
+    target_chain_cost = target_chain_params[2] * target_chain_params[1]
+    min_lp_fee = (
+        (source_chain_cost + target_chain_cost)
+        * token_params[1]
+        * (1_000_000 + lp_margin_ppm)
+        / 1e30
+    )
+
+    assert min_lp_fee == request_manager.minLpFee(ape.chain.chain_id, 999, token.address)
```

Comment:
Nitpick: since this is concerned with the fees and this file is already too big, we should move this to `test_request_manager_fees.py`.